src/cms/models/languages/language_tree_node.py

```diff
@@ -118,26 +118,72 @@ def depth(self):
     # Explicitly define functions to show documentation of base model
     @raise_if_unsaved
     def get_ancestors(self, ascending=False, include_self=False):
+        """
+        This function gets ancestors of a language tree node
+
+        :param ascending: returns whether or not the filter is ascending
+        :type ascending: bool
+
+        :param include_self: returns whether or not self is included
+        :type include_self: bool
+
+        :return: The ancestors of this language tree node
+        :rtype: ~django.db.models.query.QuerySet [ ~cms.models.languages.language_tree_node.LanguageTreeNode ]
+        """
         return super().get_ancestors(ascending, include_self)
 
     # pylint: disable=useless-super-delegation
     @raise_if_unsaved
     def get_family(self):
+        """
+        This function gets the family of a language
+
+        :return: The family of this language tree node
+        :rtype: ~django.db.models.query.QuerySet [ ~cms.models.languages.language_tree_node.LanguageTreeNode ]
+        """
         return super().get_family()
 
     @raise_if_unsaved
     def get_children(self):
+        """
+        This function gets the children of a language
+
+        :return: The children of this language tree node
+        :rtype: ~django.db.models.query.QuerySet [ ~cms.models.languages.language_tree_node.LanguageTreeNode ]
+        """
+
         return super().get_children()
 
     @raise_if_unsaved
     def get_descendants(self, include_self=False):
+        """
+        This function gets the descendants of a language
+
+        :param include_self: returns whether or not descendants should be included
+        :type include_self: bool
+
+        :return: The descendants of this language tree node
+        :rtype: ~django.db.models.query.QuerySet [ ~cms.models.languages.language_tree_node.LanguageTreeNode ]
+        """
         return super().get_descendants(include_self)
 
     def get_descendant_count(self):
+        """
+        This function counts the descendants of a language
+
+        :return: The number of descendants of this language tree node
+        :rtype: int
+        """
         return super().get_descendant_count()
 
     @raise_if_unsaved
     def get_root(self):
+        """
+        This function gets the root of the language tree
+
+        :return: The root of this language tree
+        :rtype: ~cms.models.languages.language_tree_node.LanguageTreeNode
+        """
         return super().get_root()
 
     # pylint: disable=useless-super-delegation
```

Comment:
```suggestion
```
Initially I added these methods so the docstrings of the parent class will be shown in this model's documentation. But this is no longer the case when we overwrite the docstrings, so I propose to delete them altogether. Since the docstrings of the parent class is still visible if somebody opens the "extended view" (in the top right corner), I think this is fine.